README.md

```diff
@@ -8,8 +8,8 @@
 # ⏱ metrics-process
 
 This crate provides a [Prometheus]-style [process metrics] collector for the
-[metrics] crate, supporting Linux, macOS, and Windows. The collector code is
-manually rewritten in Rust from the official Prometheus client for Go
+[metrics] crate, supporting Linux, macOS, Windows, and FreeBSD. The collector
+code is manually rewritten in Rust from the official Prometheus client for Go
```

Comment:
Is this also the case for your FreeBSD addition? If not, maybe "Some collector code..."?